<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>785</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="font">
   <font>
    <family>Monaco</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QPushButton" name="calcButton">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>10</y>
     <width>241</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Helvetica</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QLabel" name="label_req_cnt">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Время моделирования</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_time">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1000</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>270</y>
     <width>1431</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="textElideMode">
    <enum>Qt::ElideMiddle</enum>
   </property>
   <property name="showGrid">
    <bool>true</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>90</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <row>
    <property name="text">
     <string>Новая строка</string>
    </property>
   </row>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Новый столбец</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>New Column</string>
    </property>
   </column>
   <item row="0" column="0">
    <property name="text">
     <string>№ опыта</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="1">
    <property name="text">
     <string>x0</string>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="2">
    <property name="text">
     <string>x1</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="3">
    <property name="text">
     <string>x2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="4">
    <property name="text">
     <string>x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="5">
    <property name="text">
     <string>x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="6">
    <property name="text">
     <string>x1x2</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="7">
    <property name="text">
     <string>x1x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="8">
    <property name="text">
     <string>x1x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="9">
    <property name="text">
     <string>x2x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="10">
    <property name="text">
     <string>x2x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="11">
    <property name="text">
     <string>x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="12">
    <property name="text">
     <string>x1x2x3</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="13">
    <property name="text">
     <string>x1x2x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="14">
    <property name="text">
     <string>x1x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="15">
    <property name="text">
     <string>x2x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="16">
    <property name="text">
     <string>x1x2x3x4</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="17">
    <property name="text">
     <string>Y</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="18">
    <property name="text">
     <string>Yл</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="19">
    <property name="text">
     <string>Yчн</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="20">
    <property name="text">
     <string>|Y-Yл|</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="0" column="21">
    <property name="text">
     <string>|Y-Yчн|</string>
    </property>
    <property name="textAlignment">
     <set>AlignCenter</set>
    </property>
   </item>
   <item row="1" column="0">
    <property name="text">
     <string>1</string>
    </property>
   </item>
  </widget>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>1431</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <layout class="QGridLayout" name="gridLayout_7">
    <item row="1" column="0">
     <widget class="QLabel" name="label_b_8">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Частично нелинейное уравнение</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_a_8">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string notr="true">Линейное уравнение</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="line_edit_nonlin_res">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="line_edit_lin_res">
      <property name="font">
       <font>
        <family>Helvetica</family>
        <pointsize>13</pointsize>
       </font>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>10</y>
     <width>161</width>
     <height>151</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Добавить точку</string>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>141</width>
      <height>131</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_gen_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_12">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x3</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_a_9">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_gen_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_7">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x2</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_pm_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_a_13">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">x4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_edit_pm_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QPushButton" name="checkButton">
   <property name="geometry">
    <rect>
     <x>950</x>
     <y>170</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Helvetica</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Добавить точку</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>651</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QWidget" name="gridLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>631</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="1" column="0">
      <widget class="QLabel" name="label_a_2">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Разброс интенсивности генератора:        min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QLineEdit" name="line_edit_max_gen_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.09</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_a">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Интенсивность генератора:       min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_a_4">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Интенсивность ОА:       min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_b_2">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="3">
      <widget class="QLineEdit" name="line_edit_max_pm_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="line_edit_min_gen_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.001</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QLabel" name="label_b">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">max</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QLineEdit" name="line_edit_max_gen_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.1</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_a_3">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">Разброс интенсивности ОА:       min</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="line_edit_min_pm_int">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.3</string>
       </property>
      </widget>
     </item>
     <item row="3" column="2">
      <widget class="QLabel" name="label_b_4">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">max</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QLabel" name="label_b_3">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string notr="true">max</string>
       </property>
      </widget>
     </item>
     <item row="3" column="3">
      <widget class="QLineEdit" name="line_edit_max_pm_var">
       <property name="font">
        <font>
         <family>Helvetica</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0.4</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
